hashtagmap-web/front/src/components/main-page/Slider.vue

```diff
@@ -1,50 +1,50 @@
 <template>
-  <v-content>
-    <carousel :autoplay=autoplay :perPage=perPage class="carousel">
+  <v-main>
+    <carousel :autoplay="autoplay" :perPage="perPage" class="carousel">
       <slide class="carousel-component">
         <div class="image-wrap">
-          <img class="img-content" src="../../assets/mainpage/content1.png"/>
+          <img class="img-content" src="../../assets/mainpage/content1.png" />
         </div>
       </slide>
       <slide class="carousel-component">
         <div class="image-wrap">
-          <img class="img-content" src="../../assets/mainpage/content2.png"/>
+          <img class="img-content" src="../../assets/mainpage/content2.png" />
         </div>
       </slide>
       <slide class="carousel-component">
         <div class="image-wrap">
-          <img class="img-content" src="../../assets/mainpage/content3.png"/>
+          <img class="img-content" src="../../assets/mainpage/content3.png" />
         </div>
       </slide>
       <slide class="carousel-component">
         <div class="image-wrap">
-          <img class="img-content" src="../../assets/mainpage/content4.png"/>
+          <img class="img-content" src="../../assets/mainpage/content4.png" />
         </div>
       </slide>
       <slide class="carousel-component">
         <div class="image-wrap">
-          <img class="img-content" src="../../assets/mainpage/content5.png"/>
+          <img class="img-content" src="../../assets/mainpage/content5.png" />
         </div>
       </slide>
     </carousel>
-  </v-content>
+  </v-main>
 </template>
 <script>
-  import {Carousel, Slide} from "vue-carousel";
+import { Carousel, Slide } from "vue-carousel";
 
-  export default {
-    name: "Slider",
-    components: {
-      Carousel,
-      Slide,
-    },
-    data() {
-      return {
-        autoplay: true,
-        perPage: 1,
-      };
-    },
-  };
+export default {
+  name: "Slider",
+  components: {
+    Carousel,
+    Slide,
+  },
+  data() {
+    return {
+      autoplay: true,
+      perPage: 1,
+    };
+  },
+};
 </script>
 
 <style>
```

Comment:
```suggestion
<style scoped>
```

스코프 지정으로 제약하면 문제 있을까요?